utils/has.ts

```diff
@@ -1,8 +1,8 @@
 import { Obj, Curry2 } from "./types.ts";
 import curryN from "./curry_n.ts";
 
-function has(obj: Obj, prop: number | string) {
+function has(obj: Obj | 'callee', prop: number | string) {
```

Comment:
what is this for?

Reply:
it was used inside the isArguments function in is.ts

callee is a property of the arguments object. It can be used to refer to the currently executing function inside the function body of that function. This is useful when the name of the function is unknown, such as within a function expression with no name (also called "anonymous functions"). ~MDN